### PR TITLE
[docs] Clean up `public/static` folder

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Closes ENG-1728

# How

- Removes unused image files
- Removes unused video files
- Optimizes image files
- Update outdated images in Expo accounts > Account types doc

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — APPROVED** [review]
tested changes locally, left one nit, but overall LGTM! 👍

File: docs/public/static/images/accounts/converting-personal-account.jpg
[the remark this answers is not in the document]
Comment (amandeepmittal) [reply docs/public/static/images/accounts/converting-personal-account.jpg:1]: I didn't notice that. Thanks for the heads up! Changing it now.

[the remark this answers is not in the document]
Comment (amandeepmittal) [reply docs/public/static/images/accounts/converting-personal-account.jpg:1]: Done: 
CleanShot 2023-07-17 at 18 38 13@2x

[the remark this answers is not in the document]
Comment (Simek) [reply docs/public/static/images/accounts/converting-personal-account.jpg:1]: Thank you! 🙏

**Pull request merged**